internal/stack/_static/local-services-docker-compose.yml.tmpl

```
@@ -40,7 +40,7 @@ services:
     - "host.docker.internal:host-gateway"
 
   fleet-server_is_ready:
-    image: tianon/true:multiarch
+    image: "${ISREADY_IMAGE_REF}"
```

Here in this docker-compose file template, it can be used the function `fact` to get the value of the `is_ready` docker image for those 3 containers instead of the environment variable to follow the same pattern as the other services:

```suggestion
    image: "{{ fact "isready_image" }}"
```

Other docker-compose files must keep using the environment variable.